### DISCUSSION TITLE
shift key = 16 dont work?

### DISCUSSION THREAD

**Discussion by stvnkng** [cafali/SnapKey#57] [Q&A]
left shift key code?

**Comment by minteeaa** [comment]
the keycode for `LSHIFT` is `160` - as you've just brought to my attention this currently doesn't work due to my typecast that forces the `targetKey` to a `char`, of which `160` will return `0` when mapped to a `scancode`

hotfix for this incoming

**Reply by cafali** [reply]
I’ll wait to release the new version with the ALT fix until you’re done.
